### PR TITLE
feat: overview page ui

### PR DESCRIPTION
**Pull request by kushagrasarathe**
- overview page ui components with mock data

### CODE REVIEW

File: src/home/Home.jsx [hunk 1]
```diff
@@ -1,3 +1,256 @@
...
+              Active Operators
+            </div>
+            <div className="text-2xl font-normal text-white">240</div>
+          </div>
+        </Card>
+        <div className="w-2/5">
+          <CTAs />
+        </div>
+      </div>
+      <div className="flex items-stretch justify-between gap-4">
```
Comment (vincenthongzy) [inline src/home/Home.jsx:121]: can we set `sm:flex-col` for smaller screens?

File: src/home/OverviewLRTDistribution.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            ) : (
+              <>
+                <Text textAnchor="middle" fill="#fff" fontSize={24} dy={0}>
+                  {`$${total}`}
+                </Text>
+
+                <Text textAnchor="middle" fill="#888" fontSize={20} dy={40}>
+                  {`${coins.length} Assets`}
+                </Text>
+              </>
```
[the remark this answers is not in the document]
Comment (kushagrasarathe) [reply src/home/OverviewLRTDistribution.jsx:93]: Yes the component is not yet finished, I'll update it along with the bar graph which i beside this pie chart.

File: src/home/OverviewLRTDistribution.jsx
[the remark this answers is not in the document]
Comment (kushagrasarathe) [reply src/home/OverviewLRTDistribution.jsx:1]: yes I'm working on it

File: tailwind.config.js [hunk 3]
```diff
@@ -41,11 +41,17 @@ export default {
...
+              2: '#7a86a5'
             },
             outline: '#293041',
-            secondary: 'orange'
+            secondary: 'orange',
+            cinder: {
+              1: '#2D344D',
+              2: '#D0D5E7',
+              3: '#A7B2D2'
+            }
```
[the remark this answers is not in the document]
Comment (kushagrasarathe) [reply tailwind.config.js:54]: yes, the colors are from the designs

File: src/home/Home.jsx [hunk 4]
```diff
@@ -1,3 +1,256 @@
...
+    if (value >= 1_000_000) {
+      return `${(value / 1_000_000).toFixed(1)}m`;
+    }
+
+    if (value >= 1_000) {
+      return `${(value / 1_000).toFixed(1)}k`;
+    }
+  }
+
+  return numberFormatter.format(value);
```
[the remark this answers is not in the document]
Comment (kushagrasarathe) [reply src/home/Home.jsx:255]: yeah, will move in this one too, thanks for reminding

File: src/home/Home.jsx [hunk 5]
```diff
@@ -1,3 +1,256 @@
...
+
+const CTAs = () => {
+  return (
+    <div className="space-y-2">
+      <Card
+        radius="md"
+        className="bg-cinder-1 hover:bg-cinder-1/80 cursor-pointer flex flex-row items-center justify-normal gap-3 px-4 py-2.5"
+      >
+        <div>
+          <img src="/public/nethermind.png" />
```
Comment (vincenthongzy) [inline src/home/Home.jsx:207]: It was always be relative to the configured static asset dir
```suggestion
          <img src="/ethermind.png" />
```

**Pull request merged**